projects/storefrontstyles/scss/cxbase/blocks/forms.scss

```diff
@@ -173,7 +173,7 @@ input[type='radio'] {
   position: var(--cx-position, relative);
   display: var(--cx-display, inline-block);
   height: var(--cx-height, 40px);
-  width: var(--cx-width, 88px);
+  width: var(--cx-width, 78px);
```

Comment:
Does it have impacts for other functions?

Reply:
This correct the css for toggle switch, the original is incorrect. And there is no code use this toggle yet.